GetRecd/Controllers/ProfileSettingsViewController.swift

```diff
@@ -0,0 +1,226 @@
+            logOut()
+        }
+        
+        if indexPath.section == 3, indexPath.row == 2 {
+            print("DELETE ACCOUNT PRESSED")
+            
+            let alert = UIAlertController(title: "Deleting Account", message: "Are you sure you want to delete your account? This cannot be undone!", preferredStyle: .alert)
+            let yes = UIAlertAction(title: "Yes", style: .destructive, handler: { (action) in
+                self.deleteAccount()
+            })
+            
+            let no = UIAlertAction(title: "No", style: .default, handler: { (action) in
+                alert.dismiss(animated: true, completion: nil)
+            })
+            
+            alert.addAction(yes)
+            alert.addAction(no)
+            self.present(alert, animated: true, completion: nil)
+        }
+    }
+    
+    
+    @IBAction func editPicturePressed(_ sender: Any) {
+        DispatchQueue.main.async {
+            self.imagePicker = UIImagePickerController()
+            self.imagePicker.delegate = self
+            self.imagePicker.sourceType = UIImagePickerControllerSourceType.photoLibrary
+            self.imagePicker.allowsEditing = true
+            self.present(self.imagePicker, animated: true, completion: nil)
+        }
+    }
+    
+    func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [String : Any]) {
+        guard let selectedImage = info[UIImagePickerControllerEditedImage] as? UIImage else {
+            print("A VALID IMAGE WAS NOT SELECTED")
+            return
+        }
+        
+        DispatchQueue.main.async {
+            self.profilePicture.image = selectedImage
+        }
+        
+        profilePictureImage = selectedImage
+        guard let profileImage = profilePictureImage, let imageData = UIImageJPEGRepresentation(profileImage, 0.2) else {
+            return
+        }
+        
+        let imageUID = NSUUID().uuidString
+        let metaData = StorageMetadata()
+        metaData.contentType = "image/jpeg"
+        
+        DataService.instance.REF_PROFILE_PICS.child(imageUID).putData(imageData, metadata: metaData) { (metaData, error) in
+            
+            if error != nil {
+                print("IMAGE UPLOAD ERROR: Image wasn't uploaded to Firebase")
+            } else {
+                print("IMAGE UPLOAD SUCCESS: Image was uploaded to Firebase")
+                guard let imageURL = metaData?.downloadURL()?.absoluteString else {
+                    return
+                }
+                
+                self.profilePictureURL = imageURL
+                print("IMAGE URL: \(String(describing: self.profilePictureURL))")
+                
+            }
+        }
+        
+        DispatchQueue.main.async {
+            self.imagePicker.dismiss(animated: true, completion: nil)
+        }
+    }
+    
+    
+    /*
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
```

Comment:
Is there a reason we need all these commented out table view functions? If we will not be using them in a future sprint, they should be removed.

Reply:
It's TableViewVC boilerplate that's generated when you create the class. I'll delete it for now